build.cake

```diff
@@ -318,13 +321,23 @@ Task("Publish-HomeBrew")
     Information("Hash for creating HomeBrew PullRequest: {0}", hash);
 });
 
+Task("Publish-GitHub-Release")
```

Comment:
@patriksvensson yes, I have been testing this out on the ReSharperReports project, which I am moving to a fully Cake build script, and it has been working over there.